python/python_quiz_french.md

```diff
@@ -0,0 +1,162 @@
+I've translated the Python programming questions and answers into French:
```

Comment:
pls check file name to `python-quiz-fr.md` and if possible add flag to Readme.md